commune_app/models.py

```diff
@@ -1 +1,3 @@
 from commune_app.all_models.users import User  # noqa:F401
+from commune_app.all_models.Chore import Chore  # noqa:F401
+from commune_app.all_models.Vote import Vote  # noqa:F401
```

Comment:
please use lowercase for file name
```suggestion
from commune_app.all_models.vote import Vote  # noqa:F401
```